src/components/select-field/BaseSelectField.js

```diff
@@ -434,6 +509,14 @@ class BaseSelectField extends React.Component<Props, State> {
             selectOptions.splice(separatorIndex + index, 0, <li key={`separator${separatorIndex}`} role="separator" />);
         });
 
+        if (selectOptions.length === 0) {
```

Comment:
we know the list is empty before we run the `.map` call, so we can exit early in that case. You can use `filteredOptions`

also, this will appear in the same slot as a normal dropdown item, correct? are you producing a dom tree here that differs? this is just some DIV with a non-suit, non-component-specific class name. This should look like a disabled data item using the regular containers